tools/font-edit/twin-fedit.h

```diff
@@ -23,11 +23,8 @@
 #ifndef _TWIN_FEDIT_H_
 #define _TWIN_FEDIT_H_
 
-#include <X11/Xatom.h>
-#include <X11/Xlib.h>
-#include <X11/Xutil.h>
-#include <X11/keysym.h>
-#include <cairo-xlib.h>
+#include <SDL2/SDL.h>
```

Comment:
Only include the headers you need. Minimize the header inclusions in header files.